crates/ide_completion/src/completions/keyword.rs

```diff
@@ -48,6 +48,10 @@ pub(crate) fn complete_expr_keyword(acc: &mut Completions, ctx: &CompletionConte
         cov_mark::hit!(no_keyword_completion_in_record_lit);
         return;
     }
+    if ctx.attribute_under_caret.is_some() {
+        cov_mark::hit!(no_keyword_completion_in_record_lit);
```

Comment:
wrong cov mark?

Reply:
Hm, this should have been caught by a test...

Reply:
fwiw I forgot to write the `cov_mark` for this branch in general. I only copy pasted the if from before so that included the incorrect hit without much thought